stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -155,7 +155,12 @@ internal class PaymentSheetViewModel internal constructor(
     }
 
     internal enum class TransitionTarget {
-        AddCard
+        // User has saved PM's and is selected
+        SelectSavedPaymentMethod,
+        // User has saved PM's and is adding a new one
+        AddCard,
+        // User has no saved PM's
+        EnterPaymentMethod
```

Comment:
What's the difference between `EnterPaymentMethod` and `AddCard`? Specifically `Enter` vs `Add`?

Reply:
There's currently 2 differences. With `EnterPaymentMethod` we:
1. don't update the back stack (i.e. tapping back exits the sheet rather than transitioning back to LoadingFragment)
2. don't animate (still nailing doing the animations though)

In the future we will also have more differences
3. We will also show the Google Pay button if available.
4. AddCard will focus the CardInputWidget automatically and expand the sheet to full screen
5. The labels will likely be different

| EnterPaymentMethod | AddCard |
| - | - |
| <img width="327" alt="Screen Shot 2020-10-15 at 10 12 17 AM" src="https://user-images.githubusercontent.com/47332718/96163775-295e5800-0ecf-11eb-9cad-5fd2def717a5.png"> | <img width="305" alt="Screen Shot 2020-10-15 at 10 12 42 AM" src="https://user-images.githubusercontent.com/47332718/96163814-3a0ece00-0ecf-11eb-98a7-61a8fe2389a0.png"> |

Definitely would appreciate any ideas you have for better naming here though!